Jenkinsfile

```diff
@@ -304,6 +304,42 @@ pipeline {
                     }
                 }
 
+                stage('Hip Normal Find Mode Release') {
```

Comment:
[Recommendation] I guess it is worth to mention that only convolutions are under testing in this stage.